Website/Packages/datax-pipeline/src/modules/flowDefinition/components/flowDefinitionPanel.jsx

```diff
@@ -782,18 +788,18 @@ const mapStateToProps = state => ({
     inputValidated: Selectors.validateFlowInput(state),
     referenceDataValidated: Selectors.validateFlowReferenceData(state),
     functionsValidated: Selectors.validateFlowFunctions(state),
-    queryValidated: Selectors.validateFlowQuery(state),
     scaleValidated: Selectors.validateFlowScale(state),
     outputsValidated: Selectors.validateFlowOutputs(state),
     rulesValidated: Selectors.validateFlowRules(state),
-    flowValidated: Selectors.validateFlow(state)
+    flowValidated: Selectors.validateFlow(state),
+    queryValidated: QuerySelectors.validateQueryTab(state)
 });
 
 // Dispatch Props
 const mapDispatchToProps = dispatch => ({
     // Init Actions
     initFlow: context => dispatch(Actions.initFlow(context)),
-
+    initQuery: query =>dispatch(QueryActions.initQuery(query)),
```

Comment:
nit: put a space between => and dispatch

Reply:
prettifier should've taken care of this, did you not open the datax-pipeline package folder when doing the edits?

Reply:
updated. Prettifier is not working.